internal/v1/handler_post_compose_test.go

```diff
@@ -2821,6 +2821,124 @@ func TestComposeCustomizations(t *testing.T) {
 				},
 			},
 		},
+		// content template with 1 custom repository
+		{
+			imageBuilderRequest: v1.ComposeRequest{
+				Customizations: &v1.Customizations{
+					Template: &mocks.TemplateID,
+				},
+				Distribution: "rhel-8",
+				ImageRequests: []v1.ImageRequest{
+					{
+						Architecture: "x86_64",
+						ImageType:    v1.ImageTypesGuestImage,
+						UploadRequest: v1.UploadRequest{
+							Type:    v1.UploadTypesAwsS3,
+							Options: uo,
+						},
+						SnapshotDate: common.ToPtr("2000-01-30T00:00:00Z"),
+					},
+				},
```

Comment:
I think specifying a template and a snapshot date shouldn't be allowed. It's either/or, as the date is already present in the template.

Reply:
that makes sense, i added it to the compose request here since i was extracting the date from the template on creation of a blueprint and adding it to the image request. but that's probably not necessary